net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Web server vulnerability scanner"
+HOMEPAGE="http://www.cirt.net/Nikto2"
+COMMIT="fb2655b8316097723d4977c792817a21117b2736"
+MY_P="${PN}-${COMMIT}"
+SRC_URI="https://github.com/sullo/nikto/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~x86-macos"
+IUSE=""
+
+# nikto provie its own libwhisker, do no use net-libs/libwhisker[ssl]
+# https://bugs.gentoo.org/533900
+RDEPEND="
+	dev-lang/perl
+	dev-perl/Net-SSLeay
+	net-analyzer/nmap
+	virtual/perl-JSON-PP
+"
+DEPEND=""
```

Comment:
We can drop this empty line, see [here](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-620007.3).